docs/source/publish_and_share_kedro_viz_on_azure.md

```diff
@@ -0,0 +1,121 @@
+
+![](./images/azure_add_role_assign.png)
+
+![](./images/azure_storage_role.png)
+
+On the Members tab, select User, group, or service principal to assign the selected role to the app registration. Click on Select members, and find your app registration name by typing in the Select box as shown below 
+
+![](./images/azure_member_assign.png)
+
+## Step 3: Set credentials
+
+Once that's completed, you'll need to set your Azure credentials as environment variables in your terminal window, as shown below:
+
+```bash
+export AZURE_STORAGE_TENANT_ID="your-app-tenant-id"
+export AZURE_STORAGE_CLIENT_ID="your-app-client-id"
+export AZURE_STORAGE_CLIENT_SECRET="your-app-client-secret-value"
+```
+
+For more information, see the official Azure documentation about [how to work with environmental credentials](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential?view=azure-dotnet).
+
+```{note}
+Kedro-Viz uses Gen2 filesystem protocol `abfs` to write files on AzureBlobStorage.
+```
+
+```{important}
+Having a `$web` container in your AzureBlobStorage is mandatory to use Kedro-Viz publish and share feature on Azure. For more information, see the official Azure documentation about 
+[Setting up a static website](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website#setting-up-a-static-website).
+```
+
+### Permissions and access control
+
+All permissions and access control are controlled by Azure. It's up to you, the user, if you want to allow anyone to see your project or limit access to certain IP addresses, users, or groups.
+
+You can control who can view your visualisation using [attribute-based access control](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-auth-abac). See the official Azure documentation for more information.
+
+### Billing
+
+You pay for storing objects on your AzureBlobStorage. The amount you pay depends on the volume of data stored per month, quantity and types of operations performed, along with any data transfer costs, data redundancy option selected.
+
+See the official [Azure documentation](https://azure.microsoft.com/en-us/pricing/details/storage/blobs/) for more information.
+
+
+## Step 4: Publish and share the project
+Once your Cloud storage is configured and the credentials are set, you are now ready to publish and share your Kedro-Viz project. Start Kedro-Viz by running the following command in your terminal:
+
+```bash
+kedro viz run
+```
+
+Click the **Publish and share** icon in the lower-left of the application. You will see a modal dialog to select your relevant AWS Bucket Region and enter your Bucket Name.
+
+```{note}
+From Kedro-Viz version 8.0.0, you will see a modal dialog to select your hosting platform, input your bucket name and endpoint link. The endpoint link can be found under **Storage account -> Capabilities -> Static website -> Primary endpoint**.
```

Comment:
if we choose to mention that azure support is added from 8.0.0 at the top of this page, we can modify the note to be - `You will see a modal dialog to select your hosting platform, input your bucket name and endpoint link. The endpoint link can be found under **Storage account -> Capabilities -> Static website -> Primary endpoint**.` without **From Kedro-Viz version 8.0.0,**